src/ui/CustomStyleEditor.tsx

```diff
@@ -456,10 +457,14 @@ export class CustomStyleEditor extends React.PureComponent<any, any> {
   // [FS] IRAD-1127 2020-12-31
   // to populate the selected custom styles.
   onSelectCustomStyle(e) {
-    if (null !== customStyles) {
+    if (customStyles !== null) {
```

Comment:
This should just be a truthy check. The next line will hard fail if customStyles is `undefined`